src/codegen/templates/property_deprecated_template.py.mako

```diff
@@ -0,0 +1,27 @@
+<%def name="script_deprecated_property(attributes)">
+<%
+        from codegen.utilities.attribute_helpers import get_deprecated_attributes
+        deprecated_attributes = get_deprecated_attributes(attributes)
+    %>\
+%for old_property, attribute in deprecated_attributes.items():
+        @deprecation.deprecated
+        @property
+        def ${old_property}(self):
```

Comment:
`deprecated` adds a deprecation message to the end of the function's docstring. If we were deprecating hand-written functions, we would leave the old docstrings alone, but since we're doing codegen, I don't think it's worth the effort to generate the old docstrings.

I tried it out:
```
    @property
    @deprecation.deprecated(deprecated_in="0.6", current_version=__version__, details="Use ai_strain_gage_force_read_from_chan instead.")
    def ai_strain_force_read_from_chan(self):
        return self.ai_strain_gage_force_read_from_chan

    @ai_strain_force_read_from_chan.setter
    @deprecation.deprecated(deprecated_in="0.6", current_version=__version__, details="Use ai_strain_gage_force_read_from_chan instead.")
    def ai_strain_force_read_from_chan(self, val):
        self.ai_strain_gage_force_read_from_chan = val

    @ai_strain_force_read_from_chan.deleter
    @deprecation.deprecated(deprecated_in="0.6", current_version=__version__, details="Use ai_strain_gage_force_read_from_chan instead.")
    def ai_strain_force_read_from_chan(self):
        del self.ai_strain_gage_force_read_from_chan
```

Here's the documentation output:
```
...
|  ai_sound_pressure_units
 |      :class:`nidaqmx.constants.SoundPressureUnits`: Specifies the
 |          units to use to return sound pressure measurements from the
 |          channel.
 |
 |  ai_strain_force_read_from_chan
 |      .. deprecated:: 0.6 Use ai_strain_gage_force_read_from_chan instead.
 |
 |  ai_strain_gage_cfg
 |      :class:`nidaqmx.constants.StrainGageBridgeType`: Specifies the
 |          bridge configuration of the strain gages.
 |
 |  ai_strain_gage_force_read_from_chan
 |      bool: Specifies whether the data is returned by DAQmx Read when
 |          set on a raw strain channel that is part of a rosette
 |          configuration.
 |
 |  ai_strain_gage_gage_factor
 |      float: Specifies the sensitivity of the strain gage.  Gage
 |          factor relates the change in electrical resistance to the
 |          change in strain. Refer to the sensor documentation for this
 |          value.
...
```

It would also be possible for the template to directly generate this docstring, but having `deprecated` do it causes this information to be included in any deprecation warnings (if they are enabled).